### DISCUSSION TITLE
Arc welder

### DISCUSSION THREAD

**Discussion by digital0785** [moonglow/FlashForge_Marlin#136] [Q&A]
For the current firmware listed are g2/g3 commands enabled? If not would it be possible to turn it on? it produces nice results when possible 

sorry if this has been asked I looked quick but didn't see anything.

thank you all again for everything!

**Comment by GrandWaz00** [comment]
I'm running moonglow 0.15.0 (Marlin 2.0.9.5) and it is handling G2/G3

EDIT: My printer is a Dremel 3D20

**Reply by digital0785** [reply]
awesomme! i was going to try it but on my ender it did some pretty.... uhhh .. shall we say.. "crunchy" mistakes so I figured I'd ask before possibly having it damage something LOL
